### ISSUE TITLE
Rearrange repo a bit to better "hide" vendor

### ISSUE THREAD

**Issue by gholt** [getcfs/megacfs#288]
Should put executables under their own directories and vendor there.

**Comment by gholt** [comment]
I tried this out and it just isn't really feasible. Moving everything resulting in an executable under a subdirectory and vendoring there means you also have to have in your GOPATH all the third party libraries our libraries need, as well as the vendored versions. To be honest, the only reason I wanted to do this was because there used to be problems importing libraries that had a vendor in their structures; but I'm pretty sure that's no longer a problem anyway.

tl;dr I'm closing this as "nope"
